darwin/dataset/remote_dataset.py

```diff
@@ -387,7 +387,10 @@ def add_annotation_class(self, annotation_class):
             cls
             for cls in all_classes
             if cls["name"] == annotation_class.name
```

Comment:
Might be a good idea to put this if condition inside a function that makes it clear that it does (with the correct name) to go along the lines of the Python Good code practices.

Reply:
Could you share an example code snippet you feel would make this line clearer?

To me; putting it in a function would require going into that function to see what it does; which is an extra step when trying to understand the code.
That line of code only does one very obvious thing - it compares two values.

Reply:
I see you are comparing values, but I dont understand what is the semantic meaning you are trying to convey. 
Are you trying to check the class is the same or is compatible?

Reply:
Discussed on a productive call; understand Pedro's input - leaving as is for now